code/modules/cargo/packs/security.dm

```diff
@@ -321,6 +329,13 @@
 					/obj/item/gun/energy/disabler)
 	crate_name = "disabler crate" */
 
+/datum/supply_pack/security/minigunlaser
```

Comment:
I'm a little hesitant with this one. From my understanding, this is one of the best weapons in the game.